NuGetTrends.Web.dockerfile

```diff
@@ -0,0 +1,28 @@
+# ** Build
+
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+WORKDIR /src
+
+COPY src/Directory.Build.props ./
+COPY src/NuGet.Protocol.Catalog NuGet.Protocol.Catalog
+COPY src/NuGetTrends.Data NuGetTrends.Data
+
+# Avoid copying /Portal
+COPY src/NuGetTrends.Web/Properties NuGetTrends.Web/Properties
+COPY src/NuGetTrends.Web/NuGetTrends.Web.csproj NuGetTrends.Web/NuGetTrends.Web.csproj
+COPY src/NuGetTrends.Web/*.cs NuGetTrends.Web/
+COPY src/NuGetTrends.Web/*.json NuGetTrends.Web/
+
+RUN dotnet publish NuGetTrends.Web -o NuGetTrends.Web/artifacts -c Release
+
+# ** Run
+
+FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS run
+WORKDIR /app
+
+EXPOSE 80
+EXPOSE 443
+
+COPY --from=build /src/NuGetTrends.Web/artifacts ./
+
+ENTRYPOINT ["dotnet", "NuGetTrends.Web.dll"]
```

Comment:
Like I mentioned in the comments, this docker file is used to produce our image that we use for deployment. I guess now it's broken since we don't have the `dist` folder present here. I don't think it's that problematic that the files are copied. `dotnet publish` does not include the `node_modules` folder already, so maybe we could also enhance it to exclude the rest if we want to be optimal.

Edit: I just tested and `dotnet publish` only includes the `dist` folder, so we are good. 
![image](https://user-images.githubusercontent.com/5938087/118410649-c9ac7300-b690-11eb-901b-d926bc362707.png)